### PR TITLE
Tool feature

### PR DESCRIPTION
**Pull request by nmcayouette**

### CODE REVIEW

File: FormGenerator.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// HTML Form generation tool
+// HTML Form Generation Tool
```
Comment (LukeBillington) [inline FormGenerator.js:1]: Looking great! Suggest trying to compress the various input types into a function, to reduce code repetition but the level of attributes you provide is great. Also, on the errors, consider using an error object as mentioned on FSO, it's really simple `throw new Error(message)`. Keep it up!

**Review by LukeBillington — CHANGES_REQUESTED** [review]
- Compress inputs into a function
- Use error objects

File: FormGenerator.js (hunk 1 above)
Comment (nmcayouette) [reply FormGenerator.js:1]: I updated the attribute addition into a single function to reduce code repetition. Error reporting functionality may be implemented better to come, but am going to leave as simple console for now.

**Review by LukeBillington — APPROVED** [review]

**Comment by LukeBillington** [comment]
Nice! LGTM

**Pull request merged**